tests/test_video.py

```diff
@@ -88,6 +94,17 @@ def test_can_be_downsampled_to(
     assert metadata.can_be_downsampled_to(target_metadata) == expected
 
 
+def test_video_from_image():
+    with Image.open(_Configuration.SMALL_IMG_PATH) as img:
+        img = np.array(img)
+    video = Video.from_image(img, fps=30, length=0.5)
+
+    assert video.frames.shape == (15, *img.shape)
+    assert video.fps == 30
+    assert np.array_equal(video.frames[0], img)
+    assert np.array_equal(video.frames[-1], img)
```

Comment:
Can we also visually confirm that the generated video is correct? I remember having issues with PIL flipping the axis so I would make sure by saving the video and confirming it looks correct.

Reply:
After small fix it is correct.